nxc/protocols/nfs/proto_args.py

```diff
@@ -5,6 +5,7 @@ def proto_args(parser, parents):
     dgroup = ldap_parser.add_argument_group("NFS Mapping/Enumeration", "Options for Mapping/Enumerating NFS")
     dgroup.add_argument("--shares", action="store_true", help="Authenticate locally to each target")
     dgroup.add_argument("--shares-list", action="store_true", help="Listing enumerated shares")
+    dgroup.add_argument("--bruteforce-uid", nargs="?", type=int, const=4000, metavar="MAX_UID", help="Enumerate shares by bruteforcing UIDs")
```

Comment:
to keep consistency with pro SMB with option (`--rid-brute`), maybe it should be rename `--uid-brute`

Reply:
--uid-brute sounds good! I will rename it. Also yes, it seems like SMB